ansible_base/authentication/authenticator_plugins/local.py

```diff
@@ -94,11 +94,17 @@ def _can_authenticate_from_controller(self, username, password):
         If the user is valid, update the gateway users credentials with the controller credentials.
         """
         try:
-            UserModel._default_manager.get_by_natural_key(username)
+            user = UserModel._default_manager.get_by_natural_key(username)
         except UserModel.DoesNotExist:
             logger.warning(f"User '{username}' does not exist in the database.")
             return False
 
+        # Skip controller authentication if user has use_controller_password field set to False
+        # Default to False when field doesn't exist (test environments)
+        if not getattr(user, 'use_controller_password', False):
+            logger.warning(f"User '{username}' has a usable password already.")
```

Comment:
```suggestion
            logger.warning(f"User '{username}' password not in Controller.")
```

There will be users with `use_controller_password=False` and unsuable password (LDAP users in Gateway).

Reply:
Great suggestion, updated